### PR TITLE
fix: #8 LIN data_type UINT16

### PR DESCRIPTION
**Pull request by wildintellect**
Also improved the links section with corrected media_type and additional pdf references.

Before you submit a pull request, please fill in the following:

**Related Issue(s):**
- #8

**Description:**
- Corrected the data_type for LIN files. To UINT16
- Added some additional documentation links

**PR checklist:**

- [x] Code is formatted (run `scripts/format`).
- [x] Code lints properly (run `scripts/lint`).
- [x] Tests pass (run `scripts/test`).
- [ ] Documentation has been updated to reflect changes, if applicable.
- [x] Examples have been updated to reflect changes, if applicable
- [ ] Changes are added to the [CHANGELOG](../CHANGELOG.md).

### CODE REVIEW

**Comment by wildintellect** [comment]
FYI, I hardcoded a fix for https://github.com/stactools-packages/template/issues/79 to make CodeCov pass.

File: src/stactools/palsar2_scansar/constants.py [hunk 1]
```diff
@@ -107,15 +120,15 @@
...
-            "role": ["data", "local-incidence-angle"],
+            "role": ["local-incidence-angle"],
         }
     ),
     "MSK": AssetDefinition(
         {
             "title": "mask",
             "type": pystac.MediaType.COG,
             "description": "Quality Mask",
-            "role": ["data", "data-mask"],
```
Comment (jjfrench) [inline src/stactools/palsar2_scansar/constants.py:118]: From our discussion - can add in multiple roles for the asset

Comment (wildintellect) [reply src/stactools/palsar2_scansar/constants.py:118]: now added "metadata" to the linci and mask.

File: src/stactools/palsar2_scansar/constants.py [hunk 2]
```diff
@@ -40,14 +40,27 @@
...
-        media_type="application/pdf",
-        title="PALSAR-2 ScanSAR NRB product description",
+        media_type=pystac.MediaType.HTML,
+        title="ALOS-2 PALSAR-2 ScanSAR Products",
+    ),
+    Link(
+        rel="documentation",
+        target=("PALSAR2_CARD4L_L2.2_Geotiff_Format_Description_E_20220715.pdf"),
+        media_type=pystac.MediaType.PDF,
+        title="PALSAR-2 ScanSAR NRB L2.2 Geotiff Format description",
```
Comment (jjfrench) [inline src/stactools/palsar2_scansar/constants.py]: Didn't catch this the first go around, should "description" in title be uppercase? (same for other asset)

**Review by jjfrench — APPROVED** [review]

**Pull request merged**